app/components/access-level-pill.js

```diff
@@ -23,44 +25,56 @@ export default class AccessLevelPillComponent extends Component {
 
   get isLoading() {
     return this.confirmChangeAccessLevel.isRunning
-      || this.cancelChangeAccessLevel.isRunning
-      || this.changeConfidentiality.isRunning;
-  }
-
-  get inverseConfidentiality() {
-    return !this.args.confidential;
+      || this.cancelChangeAccessLevel.isRunning;
   }
 
   get pillSkin() {
-    let modifier;
+    let modifier = 'active-future';
     if (this.args.accessLevel) {
       switch (this.args.accessLevel.uri) {
-        case CONSTANTS.ACCESS_LEVELS.PUBLIEK:
-          modifier = 'success';
+        case CONSTANTS.ACCESS_LEVELS.INTERN_SECRETARIE:
+          modifier = 'active';
+          break;
+        case CONSTANTS.ACCESS_LEVELS.MINISTERRAAD:
+        case CONSTANTS.ACCESS_LEVELS.INTERN_REGERING:
+          if (this.args.agendaIsApproved) {
+            modifier = 'active';
+          }
           break;
         case CONSTANTS.ACCESS_LEVELS.INTERN_OVERHEID:
-          modifier = 'warning';
+          if (this.args.agendaIsFinal && this.args.agendaIsReleased) {
+            modifier = 'active';
+          }
           break;
-        case CONSTANTS.ACCESS_LEVELS.INTERN_REGERING:
-          modifier = 'danger';
+        case CONSTANTS.ACCESS_LEVELS.PUBLIEK:
+          if (this.args.agendaIsFinal && this.args.agendaIsPublished) {
+            modifier = 'active';
+          }
           break;
+        default:
       }
-    } else {
-      modifier = 'default';
     }
     return modifier;
   }
 
+  get pillIcon() {
```

Comment:
Good catch! I first wrote it with a getter because the usertest branch used different icons per access level. Example here: https://usertest.kanselarij-dev.s.redpencil.io/vergadering/607436596D035D000800020C/agenda/0bbbc7e0-af6b-11ec-9799-53809d45d2be/agendapunten/0c0a8420-af6b-11ec-9799-53809d45d2be/documenten?anchor=0c0a8420-af6b-11ec-9799-53809d45d2be

But I don't know where the icons are from and I was meaning to ask Dieter, but totally forgot.